### ISSUE TITLE
Connect Fail while connect in @livekit/react-components 1.0.2.  DO NOT USE @livekit/react-components 1.0.2!!

### ISSUE THREAD

**Issue by laskdjlaskdj12** [livekit/livekit-react#101]
After Request connect to server in livekit-react with an example

I've got livekit connection error for a few seconds
"error Client initiated disconnect"

and after that show this error message
"error could not connect PeerConnection after timeout"

So I guess this can be hand shake problem with livekit react and livekit server.

How can i solve this problem?

This is my project envirorment.

## package.json
```
{
  "name": "weco_client",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@livekit/react-components": "^1.0.2",
    "livekit-client": "^1.3.3",
    "autoprefixer": "^10.4.11",
    "axios": "^0.27.2",
    "next": "^12.3.0",
    "postcss": "^8.4.16",
    "postcss-loader": "^7.0.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-textarea-autosize": "^8.3.4",
    "recoil": "^0.7.5",
    "tailwindcss": "^3.1.8",
    "ws": "^8.8.1"
  },
  "devDependencies": {
    "eslint": "8.23.0",
    "eslint-config-next": "12.2.5"
  }
}
```

## Server envirorment
livekit-server version 1.2.3

**Comment by laskdjlaskdj12** [comment]
I've temporly fixed @livekit/react-componetns version as 1.0.1

## fixed package.json
```{
  "name": "weco_client",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@livekit/react-components": "^1.0.2",
    "livekit-client": "^1.3.3",
    "autoprefixer": "^10.4.11",
    "axios": "^0.27.2",
    "next": "^12.3.0",
    "postcss": "^8.4.16",
    "postcss-loader": "^7.0.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-textarea-autosize": "^8.3.4",
    "recoil": "^0.7.5",
    "tailwindcss": "^3.1.8",
    "ws": "^8.8.1"
  },
  "devDependencies": {
    "eslint": "8.23.0",
    "eslint-config-next": "12.2.5"
  }
}
```

**Comment by davidzhao** [comment]
what are the steps to repro here? which example are you seeing the issue with?

**Comment by laskdjlaskdj12** [comment]
I'm Issued @livekit/react-components. whitch newest version (1.0.2) in npm.

**Comment by davidzhao** [comment]
Can you reproduce the issue with https://example.livekit.io? that's running the latest version of the SDK.

**Comment by laskdjlaskdj12** [comment]
I've already reproduce in this repository and it working good for me. 
so it's not different "livekit/react-components" and "livekit-client" package.json this repo and my repo. 

but however i've investigate why getting error in the same package.json? so i've find it's difference in yarn.lock file.

In yarn.lock file, this repo is only using livekit-client as below, there is no livekit-copmonents,

i guess component directory in this repo, are using  instead of download yarn remote repo.

```livekit-client@^1.3.2:
  version "1.3.2"
  resolved "https://registry.yarnpkg.com/livekit-client/-/livekit-client-1.3.2.tgz#72716de3ac51be4768498b7f42f06d8a40a6c624"
  integrity sha512-hBj4SCeB++xsewg5a1ebb6fPuqVMDlZplNOFxkrPz8TA87jXM1AhdzCQokbHVlpSgNPf1srM47/56yvjrb+eVQ==
  dependencies:
    async-await-queue "^1.2.1"
    events "^3.3.0"
    loglevel "^1.8.0"
    protobufjs "^7.0.0"
    sdp-transform "^2.14.1"
    ts-debounce "^4.0.0"
    typed-emitter "^2.1.0"
    ua-parser-js "^1.0.2"
    webrtc-adapter "^8.1.1"
```
   
but my yarn file is different

```"@livekit/react-components@1.0.1":
  version "1.0.1"
  resolved "https://registry.yarnpkg.com/@livekit/react-components/-/react-components-1.0.1.tgz#037f0f3686b7fd0279522630acb993937c658a24"
  integrity sha512-/+GGOVGmjSeqz3VOW0itJ+gnAsceJWA1AWFPDsrQSiiyVPG2uY7fdGEzrr1cEOxvKdUS6Wy5La8r0hdEANKkFg==
  dependencies:
    "@fortawesome/fontawesome-svg-core" "^6.1.1"
    "@fortawesome/free-solid-svg-icons" "^6.1.1"
    "@fortawesome/react-fontawesome" "^0.1.18"
    "@livekit/react-core" "^1.0.1"
    "@types/react-responsive" "^8.0.2"
    react-aspect-ratio "^1.0.50"
    react-responsive "^8.2.0"
    react-tiny-popover "^6.0.5"

"@livekit/react-core@^1.0.1":
  version "1.0.2"
  resolved "https://registry.yarnpkg.com/@livekit/react-core/-/react-core-1.0.2.tgz#e103b425061d609853d603a6eef2dc7ee212b18d"
  integrity sha512-aHsb7nWL0QSMx72EQ1Xb9fshNgidnafRL+CouNYoljd2XqDm1Yv6bRGijBe1gXyzhQLNzttWaBWcbwlU1/jq+A==

livekit-client@^1.3.3:
  version "1.4.1"
  resolved "https://registry.yarnpkg.com/livekit-client/-/livekit-client-1.4.1.tgz#2638273cb1753a001596c03788567c352bb496bc"
  integrity sha512-0k2Qx2cqQTr7v3Gg1Tet879b3ja1iF1BCk4oTv3n1zfAzROZvq5B1YFY29wIMVUKKXrZgL8VEkj9Fz2qcVrNOA==
  dependencies:
    async-await-queue "^1.2.1"
    events "^3.3.0"
    loglevel "^1.8.0"
    protobufjs "^7.0.0"
    sdp-transform "^2.14.1"
    ts-debounce "^4.0.0"
    typed-emitter "^2.1.0"
    ua-parser-js "^1.0.2"
    webrtc-adapter "^8.1.1"
```
 
so i delete this yarn.lock file and component directory this repo 
and execute "yarn" and "yarn start".

after that, it showing the same issue that i've write in issued
however when i using package.json (as looking above)

**Comment by laskdjlaskdj12** [comment]
so i think, fixed version is not update in npm.

**Comment by lukasIO** [comment]
hi, 
the version mismatch should be fixed in 1.0.3, feel free to reopen if the problem persists for you!
